reference/anbox-features.md

```diff
@@ -9,3 +9,100 @@ The following table lists some Anbox features and whether they are supported for
 | [Custom Android ID](https://discourse.ubuntu.com/t/ams-configuration/20872#custom-android-id-10)                    | ✓    |      |
```

Comment:
This link should be updated to point below.

Reply:
Done